### PR TITLE
added sample data

### PR DESCRIPTION
**Pull request by zoeames**
To Test:
* there should be main.js
* there should be a dinos array
* there should be a employees array
* there should be a myInfo object

### CODE REVIEW

**Comment by rousell** [comment]
ZOEEE YOU NEED AN INDEX.HTML !!!! 🤬🤬🤬🤬🤬🤬🤬🤬🤬🤬🤬

File: main.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  name: "Al Acrocanthosaurus",
+  id: 10,
+  dob: "01/11/0000",
+  bio: "NOPE NOPE NOPE",
+  imageUrl: "https://i.ebayimg.com/images/g/l3EAAOSwMQhaet~m/s-l1600.jpg",
+  isAvailable: false,
+  isCarnivore: true,
+  type: "Acrocanthosaurus"
+}];
+
```
Comment (morecallan) [inline main.js:101]: @zoeames - stop putting in all these extra spaces. jeez.

**Comment by zoeames** [comment]
@rousell This ticket specified a js file only.  The index.html file is being created elsewhere

**Comment by rousell** [comment]
@zoeames oh yeah! i see that now. you're good!

File: main.js (hunk 1 above)
Comment (zoeames) [reply main.js:101]: oops fixed it

**Comment by morecallan** [comment]
🦄 🌮 👍

**Pull request merged**